src/sentry/static/sentry/app/views/organizationRoot.jsx

```diff
@@ -0,0 +1,28 @@
+import {withRouter} from 'react-router';
+import React from 'react';
+
+import {setActiveProject} from 'app/actionCreators/projects';
+import {setLastRoute} from 'app/actionCreators/navigation';
+
+/**
+ * This is the parent container for organization-level views such
+ * as the Dashboard, Stats, Activity, etc...
+ *
+ * Currently is just used to unset active project
+ */
+class OrganizationRoot extends React.Component {
+  componentDidMount() {
+    setActiveProject(null);
```

Comment:
Do we also want to clear last route here?

Reply:
In the current state, I don't think it has an affect either way. I think we should keep `lastRoute`, since it will still be correct when this mounts (and if any of these root views every need lastRoute, it'd be accurate).